skins.lua

```diff
@@ -1,286 +1,283 @@
--- TODO: Check if the indices are backwards-compatible after "translation"
---local S = character_creator.S
-
 character_creator.skins = {
 	skin = {
-		["Fair Skin"] = "cc_skin_fair.png",
-		["Green Skin"] = "cc_skin_green.png",
-		["Red Skin"] = "cc_skin_red.png",
-		["Tanned Skin"] = "cc_skin_tanned.png",
-		["White Skin"] = "cc_skin_white.png",
-		["Black Skin"] = "cc_skin_black.png",
-		["Dark Skin"] = "cc_skin_dark.png",
+		[character_creator.NS("Fair Skin")] = "cc_skin_fair.png",
```

Comment:
This is very repetitive and clutters the code. Suggestion: `local NS = character_creator.NS` on top, then use `[NS(...`

Reply:
It's not elegant indeed.